packages/patternlab/styleguide/source/assets/js/modules/mapLeaflet.js

```diff
@@ -99,6 +98,7 @@ export default (function (window,document, $) {
     window.leafletMap = mymap;
 
     window.stopFocus = false;
+    window.stopOffFocus = false;

```

Comment:
what's stopFocus and stopOffFocus mean, can you add a comment to explain these custom variables?

Reply:
@ddiestra

Reply:
Those are not longer used, both variables have been removed